wire-ios-build-assets

```diff
@@ -1 +1 @@
-Subproject commit 9f88be88dfa910ad448efb0b4908e11dda491a6b
+Subproject commit e0600004b026a905180f071842b2812433e04800
```

Comment:
@KaterinaWire is this correct?

Reply:
since we merge it to develop, should be the last one `9f88be88dfa910ad448efb0b4908e11dda491a6b`